.github/workflows/ci_workflows.yml

```diff
@@ -85,9 +85,15 @@ jobs:
           windows: py311-test-alldeps
           posargs: --durations=50
 
-        - name: Python 3.11 with all optional dependencies (MacOS X)
+        - name: Python 3.11 with all optional dependencies (MacOS X, arm64)
           macos: py311-test-alldeps
           posargs: --durations=50 --run-slow
+          runs-on: macos-14
+
+        - name: Python 3.11 with all optional dependencies (MacOS X, x86)
+          macos: py311-test-alldeps
+          posargs: --durations=50 --run-slow
```

Comment:
We probably don't need `--durations=50 --run-slow` on both ?

Reply:
Good point. I'll update it when we know wether your idea of moving x86 to cron gets traction. Thanks !